django_cloud_deploy/tests/e2e/gae_deploy_test.py

```diff
@@ -31,7 +31,7 @@
 from selenium.webdriver.chrome import options
 
 
-class GAEDeployAndUpdateE2ETest(test_base.ResourceCleanUpTest):
+class GAEDeployAndUpdateE2ETest(test_base.ResourceCleanUp):
```

Comment:
Renaming ResourceCleanUpTest to ResourceCleanUp, because this class is inherited by GCPResourceCleanUp, and it is not a test.